vhdbuilder/packer/kube-proxy-images.json

```diff
@@ -5,19 +5,20 @@
         "downloadURL": "mcr.microsoft.com/oss/kubernetes/kube-proxy:v*",
         "amd64OnlyVersions": [],
         "multiArchVersions": [
-          "1.20.13-hotfix.20220127",
           "1.20.13-hotfix.20220210",
+          "1.20.13-hotfix.20220210.1",
           "1.20.15-hotfix.20220201",
-          "1.21.7-hotfix.20220130",
+          "1.20.15-hotfix.20220201.1",
           "1.21.7-hotfix.20220204",
-          "1.21.9-hotfix.20220130",
+          "1.21.7-hotfix.20220310.1",
           "1.21.9-hotfix.20220204",
+          "1.21.9-hotfix.20220310.1",
           "1.22.4-hotfix.20220201",
-          "1.22.4-hotfix.20220223",
+          "1.22.4-hotfix.20220310.1",
```

Comment:
what's the version currently using?
1.22.4-hotfix.20220201
or 
1.22.4-hotfix.20220223

Reply:
and for other versions.

Reply:
the former, and confirmed removed versions are not in use
```
		"v1.20.13": "v1.20.13-hotfix.20220210",
		"v1.20.15": "v1.20.15-hotfix.20220201",
		"v1.21.7":  "v1.21.7-hotfix.20220204",
		"v1.21.9":  "v1.21.9-hotfix.20220204",
		"v1.22.4":  "v1.22.4-hotfix.20220201",
		"v1.22.6":  "v1.22.6-hotfix.20220130",
		"v1.23.3":  "v1.23.3-hotfix.20220130.2",
```